script/fix_claim_reference.rb

```diff
@@ -0,0 +1,16 @@
+require File.expand_path('../../config/environment',  __FILE__)
+
+
+Card::Auth.as_bot do
+
+  claim_cards = Card.search :type_id=>Card::ClaimID,:not=>{:right_plus=>[Card[:source].name,{:refer_to=>{:type_id=>Card::SourceID} }]}
+  claim_cards.each do |card|
+    claim_source = card.fetch :trait=>:source
+    if claim_source
+      puts claim_source.name
```

Comment:
the above is fine, but I think you could also do 
````
Card.search :right=>Card[:source].name, :not=>{:refer_to=>....
````
then you would have all the +source cards.

Doesn't matter really, just thought you might find it interesting.